# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-text/parsing/white-space-shorthand-text-wrap.html.ini
# Commit: eb0b125d3214
# Full Hash: eb0b125d3214a04c293cbe2f21184e85ae00302a
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2023-09-28 21:51:27
# Regressor Bug: 1731541
# File Overlap Count: 1
# Description:
#   Bug 1731541 - Implement text-wrap: balance for nsBlockFrame reflow. r=emilio
#   
#   A simple form of balance for short blocks, implemented by incrementally reducing
#   the effective inline-size used during line-breaking, up to the point where an
#   extra line would be created.
# ==============================================================================

diff -r af42a433722a -r eb0b125d3214 testing/web-platform/meta/css/css-text/parsing/white-space-shorthand-text-wrap.html.ini
--- a/testing/web-platform/meta/css/css-text/parsing/white-space-shorthand-text-wrap.html.ini	Thu Sep 28 17:28:18 2023 +0000
+++ b/testing/web-platform/meta/css/css-text/parsing/white-space-shorthand-text-wrap.html.ini	Thu Sep 28 17:28:19 2023 +0000
@@ -1,15 +1,6 @@
 [white-space-shorthand-text-wrap.html]
-  [`text-wrap: balance` should be set]
-    expected: FAIL
-
-  [`text-wrap` should not be affected by previous `white-space`]
-    expected: FAIL
-
   [`white-space` should overwrite previous `text-wrap`]
     expected: FAIL
 
-  [`text-wrap` should not be affected by `white-space` on the parent]
-    expected: FAIL
-
   [`white-space` should overwrite `text-wrap` on the parent]
     expected: FAIL